Bindings generation error: size_t seems to be missing on some computers

It appears that on some computers using Ubuntu 22.04 the following error occurs:
```
c_api/framebuffer.h:50:1: error: unknown type name 'size_t'
c_api/framebuffer.h:51:113: error: unknown type name 'size_t'
c_api/framebuffer.h:60:1: error: unknown type name 'size_t'
c_api/framebuffer.h:62:1: error: unknown type name 'size_t'
c_api/framebuffer.h:65:1: error: unknown type name 'size_t'
c_api/framebuffer.h:66:104: error: unknown type name 'size_t'
```
when building `libcamera-sys`.

--- a/libcamera-sys/c_api/framebuffer.h
+++ b/libcamera-sys/c_api/framebuffer.h
@@ -3,6 +3,7 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include <stddef.h>
 
 enum libcamera_frame_metadata_status {
     LIBCAMERA_FRAME_METADATA_STATUS_SUCCESS,

